lib/socks.c

```diff
@@ -327,18 +327,18 @@ CURLcode Curl_SOCKS4(const char *proxy_user,
      * Make connection
      */
     {
-      ssize_t packetsize = 9 +
+       size_t packetsize = 9 +
```

Comment:
This is bogus indentation.

Reply:
I see a threat in the following.
when the variables packetsize, hostnamelen or their sum increase beyond positive values, memory overflow to which socksreq + packetsize points can occur. however, ongoing checks will not provide protection.